pandas/io/clipboards.py

```diff
@@ -36,16 +36,15 @@ def read_clipboard(sep=r'\s+', **kwargs):  # pragma: no cover
     from pandas.io.parsers import read_csv
     text = clipboard_get()
 
-    # try to decode (if needed on PY3)
+    # try to decode (if needed)
     # Strange. linux py33 doesn't complain, win py33 does
-    if PY3:
-        try:
-            text = compat.bytes_to_str(
-                text, encoding=(kwargs.get('encoding') or
-                                get_option('display.encoding'))
-            )
-        except AttributeError:
-            pass
+    try:
+        text = compat.bytes_to_str(
+            text, encoding=(kwargs.get('encoding') or
+                            get_option('display.encoding'))
+        )
+    except AttributeError:
```

Comment:
can we remove this try/except? (and for sure the comment)

Reply:
Potentially.  Let's find out.

Reply:
https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=9911

Still need it.

Reply:
can you update the comment

Reply:
Done.